src/templates/ContentTemplate.tsx

```diff
@@ -6,58 +6,37 @@
  */
 
 import React from 'react';
-import { graphql } from 'gatsby';
 import RootLayout from 'layouts/Root';
 import { SidebarLayout } from 'layouts/Sidebar';
 import TitledLayout from 'layouts/Titled';
 import SEO from 'components/SEO';
 import { MarkdownProvider } from 'components/MarkdownProvider';
-import { MDXRenderer } from 'gatsby-plugin-mdx';
+import navigation from '../nav/content.yml';
+
+const ContentLayoutTemplate: React.FC<{ data: any; pageContext: any }> = ({
```

Comment:
I see that `data` wasn't typed before. Could you help a TS newbie out :) how should I know when these arguments (`data`, `pageContext`) don't need type definitions?

Reply:
With this content being provided from the GraphQL page query, it would require a custom type. Now that we have a consistent layout though, I've added some types to share between template pages.

There is also a `PageProps` type provided by gatsby that I used.